au.org.access-nri/model/configuration/ci/README.md

```diff
@@ -0,0 +1,14 @@
+# Model Configuration Checksums
+
+This schema is used to consolidate versions and defaults for multiple aspects of the model configuration CI pipeline, including reproducibility checks, QA checks, and scheduled checks.
```

Comment:
```suggestion
This schema is used to consolidate versions and defaults for multiple aspects of the model configuration CI pipeline, including reproducibility checks, QA checks, and scheduled checks. For more information on these tests, see  the [ACCESS-NRI/model-config-tests](https://github.com/ACCESS-NRI/model-config-tests/) repository.
```
More documentation of these files are probably going to live in `model-config-tests` eventually..